app_spec/zomes/blog/code/src/lib.rs

```diff
@@ -204,6 +217,6 @@ define_zome! {
     ]
 
     traits: {
-        hc_public [show_env, check_sum, check_send, get_sources, post_address, create_post, delete_post, delete_entry_post, update_post, posts_by_agent, get_post, my_posts,memo_address,get_memo,my_memos,create_memo,my_posts_as_committed, my_posts_immediate_timeout, recommend_post, my_recommended_posts,get_initial_post,get_history_post,get_post_with_options,get_post_with_options_latest,authored_posts_with_sources,create_post_with_agent]
+        hc_public [show_env, check_sum, check_send, get_sources, post_address, create_post, delete_post, delete_entry_post, update_post, posts_by_agent, get_post, my_posts,memo_address,get_memo,my_memos,create_memo,my_posts_as_committed, my_posts_immediate_timeout, recommend_post, my_recommended_posts,get_initial_post,get_history_post,get_post_with_options,get_post_with_options_latest,authored_posts_with_sources,create_post_with_agent,request_post_grant,get_grants]
     }
```

Comment:
Minor, but I think we should standardize whether we comma separate this list or not. Surprised the `hc-fmt` tool doesn't enforce this.

Reply:
good point.  Just added that to the soa tree.

Reply:
Also, that particular one appears to get missed by cargo fmt (I just did it manually) because its in a macro.